src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodash from 'lodash';
+
+import { Input } from 'components/bootstrap';
+import FormsUtils from 'util/FormsUtils';
+import { Button, ControlLabel, FormGroup, HelpBlock } from 'components/graylog';
+import { ColorPickerPopover } from 'components/common';
+import ColorLabel from 'components/sidecars/common/ColorLabel';
+
+class SlackNotificationForm extends React.Component {
+    static propTypes = {
+      config: PropTypes.object.isRequired,
+      validation: PropTypes.object.isRequired,
+      onChange: PropTypes.func.isRequired,
+    };
+
+    static defaultConfig = {
+      color: '#FF0000',
+      webhook_url: '',
+      channel: '#channel',
+      /* eslint-disable no-template-curly-in-string */
+      custom_message: ''
+             + 'Message: ${event.message}\n'
+             + '${if event.timerange_start}Timerange: ${event.timerange_start} to ${event.timerange_end}${end}\n'
+             + 'Streams: ${streams}\n'
+             + '${foreach event.fields field}  ${field.key}: ${field.value}${end}\n'
+             + '${if graylog_url}Graylog URL: ${graylog_url}\n${end}'
+             + '\n'
+             + '##########\n'
+             + '\n'
+             + '${if backlog}Last messages accounting for this alert:\n'
+             + '${foreach backlog message}${message}\n'
+             + '\n'
+             + '${end}${else}<No backlog>\n'
+             + '${end}',
```

Comment:
I'm not so sure about this template.  I think it might be better to have something more like the email template in the documentation: https://docs.graylog.org/en/3.3/pages/alerts.html#email-alert-notification

Reply:
I will look into this.